client/src/components/Timer.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from "react";
+import { useGame } from "@/context/GameContext";
+
+export default function Timer() {
+    
+    const { hours, minutes, seconds} = {hours: 0, minutes: 0, seconds: 0};
+    const [[hrs, mins, secs], setTime] = useState([hours, minutes, seconds]);
```

Comment:
My suggestion would be to only keep track of seconds elapsed and then create helper methods to convert the number of seconds to hh:mm:ss format.